### PR TITLE
Update grenade damage types to current usage

### PR DESCRIPTION
**Pull request by Turtlelord26**
Primarily for #493. In accordance with #491, explosive damage references unaltered.
-all instances of burn damage changed to fire damage
-frag grenade primary damage tagged as ballistic
-instances of Poison replaced with Internal
-CID grenade damage typed as Internal
-Plasma grenade damage typed as plasma

-Wording update on C10 - the reflex paragraph referenced it as a grenade but it is not a grenade.

### CODE REVIEW

File: Items/04_01_Melee_&_Thrown_Weapons.txt [hunk 1]
```diff
@@ -362,14 +362,14 @@ at the cost of one action on their next turn.
...
+Damage: D10 fire damage to exact spot of impact
 Ranged Miss Chance (to land where intended): 5 out to 10+[DEX or STR]*2 meters
 Armor Piercing Level: n/a
 Reflex Modifier: -10
 Reflex Save: 120
 
-	Creates a 7m radius cloud of burning, choking smoke. Targets take 30 Poison 
-Damage per turn in the cloud. All Poison Damage is negated by use of a gas mask.
+	Creates a 7m radius cloud of burning, choking smoke. Targets take 30 Internal 
+Damage per turn in the cloud. All Internal Damage is negated by use of a gas mask.
```
Comment (bleehu) [inline Items/04_01_Melee_&_Thrown_Weapons.txt]: Blocking; Poison damage wouldn't hurt cybernetic characters, Internal damage would.

**Review by bleehu — CHANGES_REQUESTED** [review]
One blocking issue; poison damage != internal damage in the case of cybernetic enemies.

File: Items/04_01_Melee_&_Thrown_Weapons.txt (hunk 1 above)
Comment (Turtlelord26) [reply Items/04_01_Melee_&_Thrown_Weapons.txt]: Poison is not a valid damage type. Would it be satisfactory to write in that this grenade's (non-Impact) damage can only affect biological enemies?
This also implies a new feature request for an equivalent grenade that hurts robotic enemies in the same manner.

**Review by rosebrallier — CHANGES_REQUESTED** [review]
Question: Are we going to capitalize damage types or not? I notice that Plasma and Internal are while fire and ballistic are not. We should choose a single convention.

**Comment by Turtlelord26** [comment]
Since they only appear right before the word 'damage', I think we can get away with not capitalizing them from a clarity standpoint. I don't think they qualify as terms being used outside normal meanings either, so I'll go for uncapitalized.

File: Items/04_01_Melee_&_Thrown_Weapons.txt [hunk 2]
```diff
@@ -282,36 +282,36 @@ at the cost of one action on their next turn.
...
-Damage: D10 burn damage to exact spot of impact
+Damage: D10 fire damage to exact spot of impact
 Ranged Miss Chance (to land where intended): 6 out to 10+[DEX or STR]*2 meters
 Armor Piercing Level: 0
 Reflex Modifier: -10
 Reflex Save: 110
 
 	Everyone in the room gets blinded for 2 turns. Once per turn, You can 
-attempt to break the blindness with a perception check at 85% (disregarding 
+attempt to break the blindness with a DC 85 check (disregarding
```
Comment (rosebrallier) [inline Items/04_01_Melee_&_Thrown_Weapons.txt]: What kind of check? Cause making rolls that disregard bonuses plus don't have any bonuses to disregard in the first place isn't something I've seen much in CX.

File: Items/04_01_Melee_&_Thrown_Weapons.txt [hunk 3]
```diff
@@ -282,36 +282,36 @@ at the cost of one action on their next turn.
...
+Damage: D10 fire damage to exact spot of impact
 Ranged Miss Chance (to land where intended): 6 out to 10+[DEX or STR]*2 meters
 Armor Piercing Level: 0
 Reflex Modifier: -10
 Reflex Save: 110
 
 	Everyone in the room gets blinded for 2 turns. Once per turn, You can 
-attempt to break the blindness with a perception check at 85% (disregarding 
+attempt to break the blindness with a DC 85 check (disregarding 
 bonuses). If the grenade detonates in the open, it’s blinding range is a 10m
```
Comment (rosebrallier) [inline Items/04_01_Melee_&_Thrown_Weapons.txt]: SC: "its"

File: Items/04_01_Melee_&_Thrown_Weapons.txt [hunk 4]
```diff
@@ -282,36 +282,36 @@ at the cost of one action on their next turn.
...
 
 	Everyone in the room gets blinded for 2 turns. Once per turn, You can 
-attempt to break the blindness with a perception check at 85% (disregarding 
+attempt to break the blindness with a DC 85 check (disregarding 
 bonuses). If the grenade detonates in the open, it’s blinding range is a 10m 
 radius.
 
-	Each character in the radius of a flashbang's range when it goes off may
+	Each character in the radius of a flash-bang's range when it goes off may
 make a Reflex Save of DC 110. If they succeed, they are able to shield their
```
Comment (rosebrallier) [inline Items/04_01_Melee_&_Thrown_Weapons.txt]: Sc:Is Save capitalize?

File: Items/04_01_Melee_&_Thrown_Weapons.txt [hunk 5]
```diff
@@ -282,36 +282,36 @@ at the cost of one action on their next turn.
...
 Armor Piercing Level: 1
 Reflex Modifier: -10
 Reflex Save: 100
 
-	Each Target within the blast radius takes multiple hits. 45 damage per hit, 
-automatically hits armor. Targets take full damage to the torso, legs, and arms 
-(if armor is pierced or not present): (3*45 damage)+2xBleed within a 2.5m radius.
-Only legs and torso (2*45 damage)+1xBleed within a 5m radius. At least one hit 
-must damage health to cause Bleed.
+	Each Target within the blast radius takes multiple hits. 45 ballistic
```
Comment (rosebrallier) [inline Items/04_01_Melee_&_Thrown_Weapons.txt:310]: Sc: clarity re taking multiple hits

File: Items/04_01_Melee_&_Thrown_Weapons.txt [hunk 6]
```diff
@@ -282,36 +282,36 @@ at the cost of one action on their next turn.
...
-	Each Target within the blast radius takes multiple hits. 45 damage per hit, 
-automatically hits armor. Targets take full damage to the torso, legs, and arms 
-(if armor is pierced or not present): (3*45 damage)+2xBleed within a 2.5m radius.
-Only legs and torso (2*45 damage)+1xBleed within a 5m radius. At least one hit 
-must damage health to cause Bleed.
+	Each Target within the blast radius takes multiple hits. 45 ballistic 
+damage per hit, automatically hits armor. Targets take full damage to the torso, 
+legs, and arms (if armor is pierced or not present): (3*45 damage)+2xBleed 
+within a 2.5m radius.Only legs and torso (2*45 damage)+1xBleed within a 5m 
+radius. At least one hit must damage health to cause Bleed.
```
Comment (rosebrallier) [inline Items/04_01_Melee_&_Thrown_Weapons.txt:314]: Sc: status effects capitalize?

**Review by rosebrallier — COMMENTED** [review]
I'm scope creeping all over the place so I can find these moments later.

File: Items/04_01_Melee_&_Thrown_Weapons.txt [hunk 7]
```diff
@@ -261,16 +261,16 @@ at the cost of one action on their next turn.
...
 Ranged Miss Chance (to land where intended): 5 out to 7+[DEX or STR]*2 meters
 Armor Piercing Level: 0
 Reflex Modifier: -10
 Reflex Save: 90
 
 	10m radius cloud of choking chemicals. anyone caught in the circle suffers 
 the following effects: partial blindness due to uncontrollable tears (+30% to 
-hit), coughing fits (-50% stealth), and take 10 damage per round spent in the 
-cloud. Targets can negate all effects with a gas mask. Targets who leave the 
+hit), coughing fits (-50% stealth), and take 10 Internal damage per round spent in
```
Comment (rosebrallier) [inline Items/04_01_Melee_&_Thrown_Weapons.txt]: Internal

**Review by rosebrallier — APPROVED** [review]
Yay pending bleehu

File: Items/04_01_Melee_&_Thrown_Weapons.txt (hunk 7 above)
Comment (Turtlelord26) [reply Items/04_01_Melee_&_Thrown_Weapons.txt]: ?

Comment (rosebrallier) [reply Items/04_01_Melee_&_Thrown_Weapons.txt]: Ahem. "internal damage"

File: Items/04_01_Melee_&_Thrown_Weapons.txt (hunk 2 above)
Comment (bleehu) [reply Items/04_01_Melee_&_Thrown_Weapons.txt]: Blocker: ya, we really need to know what kind of check this is.

**Review by bleehu — CHANGES_REQUESTED** [review]
Small blocking change on flashbang save. Once added, you may shipit without further review.

File: Items/04_01_Melee_&_Thrown_Weapons.txt (hunk 2 above)
Comment (Turtlelord26) [reply Items/04_01_Melee_&_Thrown_Weapons.txt]: That's a discussion issue then. The old stuff is nonfunctional enough that there are multiple ways to fix it and I'd rather not unilaterally pick one.
* DC X perception check (with Per mod)
* DC X spotting check (with spotting skill & per mod)
* DC X typeless check (i.e. no bonuses only flat roll, requires rules support in skills section of basic)
* Other

Comment (bleehu) [reply Items/04_01_Melee_&_Thrown_Weapons.txt]: Spotting or awareness seems most likely to me.

Comment (rosebrallier) [reply Items/04_01_Melee_&_Thrown_Weapons.txt]: Making it an Awareness save or a Perception check both work for me.

**Comment by Turtlelord26** [comment]
Still have to ask for a shipit, according to our rules for declared blocking issues.

**Review by rosebrallier — APPROVED** [review]
Shipit!

**Review by bleehu — APPROVED** [review]
Make shipit tha thing! Well done, Turtle!

**Pull request merged**